articles/architecture-scenarios/application/mobile-api.md

```diff
@@ -4,35 +4,290 @@ title: Mobile + API
+- __Signing Algorithm__: the algorithm to sign the tokens with. The available values are `HS256` and `RS256`. When selecting RS256 the token will be signed with the tenant's private key. For more details on the signing algorithms see the Signing Algorithms paragraph below.
+
+![Create API](/media/articles/architecture-scenarios/mobile-api/create-api.png)
+
+Fill in the required information and click the __Create__ button.
+
+<%= include('../_includes/_api-signing-algorithms.md') %>
+
+<%= include('../_includes/_api-configure-scopes.md') %>
+
+### Create the Client
+
+There are four client types in Auth0: __Native__ (used by mobile or desktop apps), __Single Page Web Applications__, __Regular Web Applications__ and __Non Interactive Clients__ (used by CLIs, Daemons, or services running on your backend). For this scenario we want to create a new Client for our Mobile Application, hence we will use Native as the client type.
+
+To create a new Client, navigate to the [dashboard](${manage_url}) and click on the [Clients](${manage_url}/#/clients}) menu option on the left. Click the __+ Create Client__ button.
+
+Set a name for your Client (we will use `Timesheets Mobile`) and select `Native` as the type.
+
+Click __Create__.
+
+![Create Client](/media/articles/architecture-scenarios/mobile-api/create-client.png)
+
+That's it for now. When we are done with the Mobile app implementation we will revisit the dashboard and this Client's settings to make some changes in its configuration.
+
+## Inside the Implementation
+
+<%= include('../_includes/_api-implement.md') %>
+
+### Implement the Mobile App
+
+In this section we will see how we can implement a Mobile application for our scenario.
+
+::: note
+[See the implementation in Android.](/architecture-scenarios/application/mobile-api/mobile-implementation-android#1-set-up-the-application)
+:::
+
+#### Authorize the User
+
+To authorize the user we will implement an [Authorization Code Grant Flow with PKCE](/api-auth/tutorials/authorization-code-grant-pkce). The mobile application should first send the user to the [authorization URL](/api/authentication#authorization-code-grant-pkce-) along with the `code_challenge` and the method used to generate it:
+
+```text
+https://${account.namespace}/authorize?
+    audience=API_AUDIENCE&
+    scope=SCOPE&
+    response_type=code&
+    client_id=YOUR_CLIENT_ID&
+    code_challenge=CODE_CHALLENGE&
+    code_challenge_method=S256&
+    redirect_uri=https://YOUR_APP/callback
+```
+
+The `GET` request to the authorization URL should include the following values:
+
+Parameter | Description
+----------|------------
+__domain__ | The value of your Auth0 Domain. You can retrieve it from the Settings of your Client at the [Auth0 Dashboard](${manage_url}/#/clients).
```

Comment:
there is no `domain` parameter. It's part of the URL and the user either reads `YOUR_AUTH0_DOMAIN` (if they are not logged in) or sth like `mpaktiti.auth0.com`